pkg/users/users.js

```diff
@@ -210,7 +210,12 @@ async function getLogins() {
         // this is impossible to parse with Date() (e.g. Firefox does not work with all time zones), so call `date` to parse it
         return cockpit.spawn(["date", "+%s", "-d", date_fields.join(' ')], { environ: ["LC_ALL=C"], err: "out" })
                 .then(out => ({ name, loggedIn, lastLogin: parseInt(out) * 1000, isLocked }))
```

Comment:
If you want to do that yourself: Just record the date from the output here, have the `catch()` branch set it to `null`, and return a single result object.

Reply:
Happy for you to do it! I'm not quite sure I'm understanding what you mean, so if you push I'll check

Reply:
Done -- can you please have a look? (Tests are still running)

Reply:
LGTM! 👍